webapp/pages/settings/index.spec.js

```diff
@@ -0,0 +1,73 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import index from './index.vue'
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:57:55Z" title="Thursday, May 2nd 2019, 7:57:55 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,73 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import index from './index.vue'
```


Oh nice, so it's possible to test Nuxt's special `page` components? How did it went? Was it big fun? :slightly_smiling_face:  @mattwr18 @ulfgebhardt